### PR TITLE
Bump react-scripts to 3.1

### PR DESCRIPTION
**Pull request by rupurt**

### CODE REVIEW

File: explorer/client/package.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "module-to-cdn": "^3.1.2",
     "prettier": "^1.18.2",
     "react-hot-loader": "^4.8.0",
-    "react-scripts": "3.0.1",
+    "react-scripts": "^3.1",
```
Comment (typescribe) [inline explorer/client/package.json]: Does it make a difference if we include the patch version? Maybe we should explicitly set it just to make it obvious

Comment (rupurt) [reply explorer/client/package.json]: I feel like we should be as permissive as possible here so we update regularly until we encounter an issue that requires us to restrict it to patch level upgrades. `react-scripts` is pretty conservative with their updates.

Comment (typescribe) [reply explorer/client/package.json]: Yeah I agree, but I meant set it to something like `^3.1.0` just to make the version obvious

Comment (rupurt) [reply explorer/client/package.json]: Gotcha. It doesn't make a difference and I don't feel strongly about either way. Updated to `3.1.0`

**Pull request merged**